buildscripts/incremental/build.sh

```diff
@@ -9,32 +9,57 @@ if [[ $(uname) == Darwin ]]; then
   # export LLVM_CONFIG explicitly as the one installed from llvmdev
   # in the build root env, the one in the bootstrap location needs to be ignored.
   export LLVM_CONFIG=$(ls $(which llvm-config))
-  # now bootstrap the toolchain for building
-  conda create -y -p ${PWD}/bootstrap clangxx_osx-64
+  
+  # Determine architecture
+  ARCH=$(uname -m)
+  if [[ "$ARCH" == "arm64" ]]; then
+    CONDA_SUBDIR=osx-arm64
+    # Bootstrap with ARM64 compiler
+    conda create -y -p ${PWD}/bootstrap clangxx_osx-arm64
```

Comment:
I think these are unused for now. Can we try M1 now?

Reply:
don't have arm on matrix currently, can remove this if not needed.

Reply:
removed